package.json

```diff
@@ -5,6 +5,8 @@
   },
   "dependencies": {
     "@dosomething/forge": "~6.6.0",
+    "babel-preset-es2015": "^6.6.0",
```

Comment:
this had to be added for elixr to work properly. see this issue: https://github.com/laravel/elixir/issues/354